README.md

```diff
@@ -177,6 +177,13 @@ If you want to manage localized fields, the ckanext-dcatapit extension requires
 
          paster --plugin=ckanext-dcatapit vocabulary load --filename path/to/license.rdf --name licenses --config=/etc/ckan/default/production.ini
 
+
+### Organization as a Rights Holder
```

Comment:
The reason of this readme update was to explain the following comment about Regions:
https://github.com/geosolutions-it/ckanext-dcatapit/issues/214#issuecomment-408831071

Why did you mention the holder and his name or identifier?

Reply:
Sorry, my bad. I wrote this just after reading comments in other issue about holder and organization.  Readme updated